wagtail_streamfield_migration_toolkit/autodetect/questioner.py

```diff
@@ -10,3 +10,7 @@ def ask_block_rename(self, old_path, new_path):
     def ask_block_remove(self, old_path):
         msg = "Was '{}' removed? [y/N]"
         return self._boolean_input(msg.format(old_path))
+
+    def ask_block_not_changed(self, old_path):
+        msg = "Is '{}' the same block as before? [y/N]"
```

Comment:
Question: do we need to give more context here? For example: `Found {block type} at path {path}, that appears to be unchanged. Is this the same block? [y/N]`

Reply:
I like this. It's the verifying threshold though here, right, so maybe the text needs to flag that there are *some* changes - we're only asking this when the blocks aren't identical. Can't quite come up with the right phrase though, will have a think!

Reply:
Maybe something like "minor differences" ?

Reply:
That sounds good to me.